### ISSUE TITLE
Is there a smaller dataset?

### ISSUE THREAD

**Issue by hmtrii** [microsoft/table-transformer#27]
Hi guys, the provided datasets are huge to my limited resource. Could you provide a smaller dataset? Thank you!

**Comment by bsmock** [comment]
This is something we can think about how to do. Can you give us more details about what you would like? Are you looking for samples from the training set or from the test set only?

**Comment by bsmock** [comment]
At a minimum I think we can and should release a few test examples to preview the dataset. Let us know if you need something more than that and we can think about what we can do.

**Comment by hmtrii** [comment]
Thanks for your reply. Yes, I need samples from the test set to review the dataset. Besides, could you provide a subset from your training set if possible? I think it should be about 10,000 images. It is helpful for researchers who don't have strong enough resources.

**Comment by mzhadigerov** [comment]
Any updates on that?

**Comment by yellowjs0304** [comment]
@bsmock Is there any update about it? I wanna just check some examples in your datasets.

**Comment by bsmock** [comment]
A little while back we put the table structure recognition dataset on [Kaggle](https://www.kaggle.com/datasets/bsmock/pubtables-1m-structure) which should give you the ability to preview and download individual samples.

I'll close this issue but let us know if you have any other issues using the dataset.

Cheers,
Brandon
